### PR TITLE
Jennyf/ui

### PR DESCRIPTION
**Pull request by jennyf19**
@henrik-me - still having issues w/the b2c tests and cannot repro it locally (long story), but thought i'd get this started at least.

### CODE REVIEW

**Comment by henrik-me** [comment]
Generally it looks great to me. Curious as to which tests are failing including how. Will stop by. Looks great.

File: tests/devapps/XForms/XForms.Android/XForms.Android.csproj [hunk 1]
```diff
@@ -127,7 +127,4 @@
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
   <!-- Import the Xamarin targets workaround -->
   <Import Project="..\Patch.Xamarin.Common.targets" />
-  <PropertyGroup>
```
Comment (MarkZuber) [inline tests/devapps/XForms/XForms.Android/XForms.Android.csproj]: why are you removing this?

File: tests/Microsoft.Identity.Test.Core.UIAutomation/CoreUiTestConstants.cs [hunk 2]
```diff
@@ -33,66 +33,59 @@ public static class CoreUiTestConstants
...
-        public const string B2CWebPasswordFacebookID = "m_login_password";
-        public const string B2CFacebookSubmitID = "u_0_5";
-        public const string GoogleAccountID = "GoogleExchange";
-        public const string WebUPNB2CGoogleInputID = "Email";
-        public const string B2CWebPasswordGoogleID = "Passwd";
-        public const string B2CGoogleNextID = "next";
-        public const string B2CGoogleSignInID = "signIn";
-        public const string B2CEditProfileContinueID = "continue";
+        public const string FacebookAccountId = "FacebookExchange";
+        public const string WebUpnB2CFacebookInputId = "m_login_email";
```
Comment (MarkZuber) [inline tests/Microsoft.Identity.Test.Core.UIAutomation/CoreUiTestConstants.cs]: #TheHeroWeDeserve  (thanks for cleaning these up)

File: tests/devapps/XForms/XForms.Android/XForms.Android.csproj (hunk 1 above)
Comment (jennyf19) [reply tests/devapps/XForms/XForms.Android/XForms.Android.csproj]: oops...deleted too much. thanks for catching this. I will update in next commit. 

---
In reply to: [246214307](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#discussion_r246214307) [](ancestors = 246214307)

File: tests/Microsoft.Identity.Test.Core.UIAutomation/CoreUiTestConstants.cs (hunk 2 above)
Comment (jennyf19) [reply tests/Microsoft.Identity.Test.Core.UIAutomation/CoreUiTestConstants.cs]: :sunglasses: 

---
In reply to: [246214528](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#discussion_r246214528) [](ancestors = 246214528)

**Comment by jennyf19** [comment]
got the tests sorted out...i decided to nuke the b2c local windows authority test for a couple reasons 1) microsoftlogin.com is deprecated 2) we have a test for b2clogin.com 3) because the login info is contained in the webview, we have to select the local account from the list, and not do the login as it's currently set up. so that would mean adding in a lot more code for a special case here, which i decided not to do and removed the test instead. I think we have enough coverage here for now. lmk.

---
In reply to: [452194486](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#issuecomment-452194486) [](ancestors = 452194486)

File: tests/devapps/XForms/XForms.Android/XForms.Android.csproj (hunk 1 above)
Comment (jennyf19) [reply tests/devapps/XForms/XForms.Android/XForms.Android.csproj]: done.

---
In reply to: [246215224](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#discussion_r246215224) [](ancestors = 246215224,246214307)

**Pull request merged**